pkg/forwarder/retry_queue_telemetry test.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Filename is wrong here, you've used a space, I don't know if `go test` is actually going through these.

Not entirely sure to know what is the purpose of this file? Is this because you're using interfaces? What's the issue using types instead?

Reply:
Updated.